doc/index.rst

```diff
@@ -35,19 +32,41 @@ interface toolkits.
    </div>
    <span class="clear_screenshots"></span>
 
-Matplotlib tries to make easy things easy and hard things possible.  You
-can generate plots, histograms, power spectra, bar charts, errorcharts,
-scatterplots, etc., with just a few lines of code.  For examples, see the
-:doc:`sample plots <tutorials/introductory/sample_plots>` and :doc:`thumbnail
-gallery <gallery/index>`.
+Matplotlib makes easy things easy and hard things possible.
 
-For simple plotting the :mod:`.pyplot` module provides a MATLAB-like interface,
-particularly when combined with IPython.  For the power user, you have full
-control of line styles, font properties, axes properties, etc, via an object
-oriented interface or via a set of functions familiar to MATLAB users.
+.. container:: bullet-box-container
+
+   .. container:: bullet-box
+
+      Create
+
+      - Easy creation of standard plots
+      - Publication quality output to common file formats
+        (png, pdf, svg, ps, ...)
+      - Interactive figures (zoom, pan, ...)
```

Comment:
standard plots is too domain specific. I think instead stealing a bit from the next line: (and trying to keep all 3 boxes in parallel structure)
```
- Develop [publication quality plots](https://matplotlib.org/gallery/index.html) with just with just a few lines of code. 
- Build [unteractive figures](https://matplotlib.org/gallery/index.html#event-handling) that can zoom, pan, update...

Reply:
ok